### DISCUSSION TITLE
Non-antagonizing weapons

### DISCUSSION THREAD

**Discussion by mefistotelis** [swfans/syndwarsfx#146] [General]
I just noticed that using flamer on a group member does not antagonize that group.

I wonder if it can be used as a strategy for some of the missions..

**Comment by Moburma** [comment]
I tried to use it to cheese the Zealot Columbo mission, but you're too far out of range, and moving close enough just triggers the enemies to attack en masse. There must be some other levels it would work...
The flamer just seems a really weird weapon, it's crap in single player, so I can only assume it was balanced for multiplayer. I used to like the flamer in the original Syndicate, it was somewhat overpowered if you got it early, but quickly became obsolete.

**Comment by mefistotelis** [comment]
Notice that if you do a constant flame on a path, enemies will stop before it. Not sure if they still do this if they're aggroed, but if they do - maybe flamer could be used to keep enemies at distance with one agent while the rest stand nearby waiting for their LR Rifles to reload.

**Comment by geist22** [comment]
Enemies (and your agents too) only react to direct damage, not to splash or explosions. It's super annoying when you're swarmed by enemies with the Launcher and your agents never shoot back themselves because they haven't received a direct hit yet but keep getting pushed around by the splash damage. But I do consider this working as intended, otherwise friendly NPCs would get antagonized by accidentally moving into your explosions.

>Zealot Columbo mission

You can exploit this behaviour there if you want and just chuck a few Nuclear Grenades into the nearby group at the start without them going hostile.

As for the Flamer, in the german version I used to play this weapon is really weird. As you know the german version is censored in many places, and one of them is that people can't be set on fire. So when you hit someone with the Flamer, one of two things happen: either they lose health incredibly fast and just die like with any other weapon, or the stream of flame goes through them and... does nothing? Like if your target had collision disabled. This was probably just a bug, but it happened very often, so the Flamer was even more useless than in the regular version because it had a good chance to do nothing at all.

-edit-
I've also noticed that some weapons have unique properties. Of particular interest is the Plasma Lance, which has two of them. For one, it deals a great deal of bonus damage to vehicles and especially buildings. You can mod the damage of other weapons to be the same or higher, but vehicles still take about half to a third of the damage, and buildings still take a good while to destroy, while the Plasma Lance does it in seconds. Perhaps this is just an additional effect of the other property, which is that the Plasma Lance has a piercing effect. If there's a straight line of NPCs in front of your agent, and you shoot at the one that's the furthest away, all of them will be hit, while with other weapons, only the closest will be hit and consumes the shot. It also works on solid terrain, if there's a wall/hill etc. that's part of the ground (not buildings), your shot will push through and hit them. On the Colombo mission, I use this admittedly exploitish trick to kill the Zealots with the Nuclear Grenades at the top of the ramps from below without aggroing them.
